src/makeChange.js

```diff
@@ -1,3 +1,63 @@
+let keys = {
+  quarters: 0,
+  dimes: 0,
+  nickels: 0,
+  pennies: 0
+}
+let takeQuarter = function(amounTGiven, pricey){
+  amounTGiven = amounTGiven - 25
+  keys.quarters++
+  return amounTGiven
+}
+let takeDime = function(amounTGiven, pricey){
+  amounTGiven = amounTGiven - 10
+  keys.dimes++
+  return amounTGiven
+}
+let takeNickel = function(amounTGiven, pricey){
+  amounTGiven = amounTGiven - 5
+  keys.nickels++
+  return amounTGiven
+}
+let takePenny = function(amounTGiven, pricey){
+  amounTGiven = amounTGiven - 1
+  keys.pennies++
+  return amounTGiven
+}
 export default function makeChange({price, amountGiven}) {
-  // your code here
+  let pricey = price
+  let results = {}
+  let amounTGiven = amountGiven
```

Comment:
You are teaching yourself bad habits with variable names like `pricey` and `amounTGiven`.